docs/rules/no-assignment-of-untracked-properties-used-in-tracking-contexts.md

```diff
@@ -69,6 +69,61 @@ class MyComponent extends Component {
 
 The autofixer for this rule will update assignments to use `set`. Alternatively, you can begin using tracked properties.
 
+## Configuration
+
+* object -- containing the following properties:
+  * array -- `extraMacros` -- array of configurations for custom macros, each with the following properties:
+    * string -- `name` -- The name the macro is exported with
+    * string -- `path` -- The file path used for importing the macro
+    * string -- `indexName` -- If this macro can also be imported through an index (like `computed` for `computed.and`), include it here
+    * string -- `indexPath` -- The path for importing the index
```

Comment:
This needs some more explanation regarding what you mean by `index`. You could say:

```
For example, with `import { computed } from '@ember/object'` and `computed.and(...)`, `@ember/object` is the `indexPath` and `computed` is the `indexName`.
```